src/System.Windows.Forms/src/System/Windows/Forms/Control.ActiveXImpl.cs

```diff
@@ -1421,8 +1421,13 @@ private Point PixelToHiMetric(int x, int y)
             /// <summary>
             ///  Our implementation of IQuickActivate::QuickActivate
             /// </summary>
-            internal HRESULT QuickActivate(Ole32.QACONTAINER pQaContainer, UnsafeNativeMethods.tagQACONTROL pQaControl)
+            internal unsafe HRESULT QuickActivate(Ole32.QACONTAINER pQaContainer, Ole32.QACONTROL* pQaControl)
```

Comment:
Why is this not `ref`?

```suggestion
            internal HRESULT QuickActivate(Ole32.QACONTAINER pQaContainer, ref Ole32.QACONTROL pQaControl)
```

Reply:
Precedent is to make these into pointers -all the other interfaces have pointers over refs in order to better handle what happens when the value is null